docs/source/reference/northstar/serversiderui.rst

```diff
@@ -0,0 +1,187 @@
+Serverside RUI
+======
+
+Server-side Rui provides a set of functions enabling servers to display complex hud elements on clients without requiring a client-side mod. These functions were introduced in Northstar ``1.10.0``.
+
+It should be noted that there’s no guarantee the client will see the hud elements.
+
+Polls
+^^^^^
+
+Creates a poll on ``player``.
+
+.. figure:: /_static/serversiderui/vote.png
+  :align: center
+  :class: screenshot
+
+  Player POV
+
+**Definition:**
+
+.. cpp:function:: void NSCreatePollOnPlayer( entity player, string header, array<string> options, float duration )
+
+**Example:**
+
+.. code-block:: javascript
+
+    void function CreateDummyPoll()
+     {
+        array<string> options = [ "Me", "You", "Both of us" ]
```

Comment:
I'd use the same text as in the example GIF to make the mapping of where which string goes clearer. ^^